### DISCUSSION TITLE
Append Right-to-Left instead of Left-to-Right

### DISCUSSION THREAD

**Discussion by deanff** [ImageMagick/ImageMagick#2470] [Help] [answered]
Is it possible to append images from right-to-left instead of left-to-right?

Example:
Image1 + Image2 = Im2Im1
Image3 + Image4 = Im4Im3
Image5 + Image6 = Im6Im5

**Comment by GeeMack** [comment, marked as answer]
If I understand the question, yes, you can read two or more images into an ImageMagick command and append them in any order. These commands all create a red square and a blue square, reverse their order, and append them horizontally to create identical outputs...

     convert -size 200x200 xc:red xc:blue +swap +append blue_red.png

     convert -size 200x200 xc:red xc:blue -reverse +append blue_red.png

     convert -size 200x200 xc:red xc:blue -insert 0 +append blue_red.png

If you're using ImageMagick v7 use "magick" instead of "convert".
